backend/core/src/migration/1641855882656-remove-application-due-time.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class removeApplicationDueTime1641855882656 implements MigrationInterface {
+  name = "removeApplicationDueTime1641855882656"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "listings" DROP COLUMN "application_due_time"`)
+  }
```

Comment:
For existing listings, we'll need to make sure the time set on application_due_date before dropping the column.